test/http-mock.ts

```diff
@@ -61,10 +61,12 @@ export function scope(basePath: BasePath, options?: nock.Options): nock.Scope {
     const body = req.requestBodyBuffers?.[0]?.toString();
 
     if (body) {
+      let jsonBody: any;
       try {
-        result.graphql = makeGraphqlSnapshot(body);
+        jsonBody = JSON.parse(body);
```

Comment:
We should check contentType for `application/json` so we don't have too many json parse failues?